### ISSUE TITLE
Version option for all languages.

### ISSUE THREAD

**Issue by kargnas** [i18next/i18next-localstorage-backend#12]
In general, we always update all of the language files when the strings are updated.
So this option will be very helpful as well:
```js
      backendOptions: {
        prefix: 'i18next_res_',
        versions: 'v1.0',
      },
```

**Comment by jamuhl** [comment]
PR welcome...

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next-localstorage-backend/pull/15 was published in v3.1.0
